io.github.garglk.Gargoyle.yaml

```
@@ -0,0 +1,30 @@
+id: io.github.garglk.Gargoyle
+runtime: org.kde.Platform
+runtime_version: "5.15-21.08"
+sdk: org.kde.Sdk
+command: gargoyle
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --filesystem=home
+  - --system-talk-name=org.freedesktop.UDisks2
+modules:
+  - name: gargoyle
+    buildsystem: cmake-ninja
+    builddir: true
+    post-install:
+      - install -D $FLATPAK_BUILDER_BUILDDIR/gargoyle-house.svg /app/share/icons/hicolor/scalable/apps/io.github.garglk.Gargoyle.svg
```

```suggestion
      - install -D ../gargoyle-house.svg /app/share/icons/hicolor/scalable/apps/io.github.garglk.Gargoyle.svg
```

Ideally this icon would be upstream

---

At the moment we have a low-res icon upstream.  I intend to include an SVG icon in the next release, but it's not yet been determined what that icon will be.  The icon here is an SVG representation of the current icon (a white house), so fits with the appearance of the current release, but may not be what ends up as the final icon.  When the next release comes out, this icon will be removed from the Flathub repo, and either added to the official release (if it's the one being used), or the release will have another SVG icon that will be suitable for Flathub.